R/geom_parallel_slopes.R

```diff
@@ -0,0 +1,168 @@
+#' Parallel slopes model
+#' 
+#' \code{geom_parallel_slopes()} fits parallel slopes model and adds its line
+#' output(s) to a \code{ggplot} object. Basically, it fits a unified model with
+#' intercepts varying between groups (which should be supplied as standard
+#' {ggplot2} grouping aesthetics: \code{group}, \code{color}, \code{fill},
+#' etc.).
```

Comment:
Could you add a comment that this geom provides functionality that `geom_smooth()` doesn't?